### PR TITLE
[Update/gift-guide]: Gift guide updates for aes 1590

### PR DESCRIPTION
**Pull request by kevin-ho87**
Updates for ticket [AES-1590](https://aesoponline.atlassian.net/browse/AES-1590)
Tweaks for Gift guide project

Product grid: 
- [x] Increased the top/bottom padding in the ‘Skin Feel’ ‘Aroma’ etc to be 20px (`<dl>`)
- [x] Increase the margin from the ATC to the Product Details from 14px to 30px
- [x] ATC Button disabled state styling with price
- [x] Tablet variant info holder min height to align CTAs
- Tablet and small desktop layout overlap seperation
- [AES-1994](https://aesoponline.atlassian.net/browse/AES-1994) - Bundles to display link instead of ATC

Product Commerce
- [x] CTA width 50%
- [x] rejig markup for responsive reordering
- [x] transition
- [x] tablet widths

General
- [x] IE fixes 
- [x] Adding `forwardRef` to both components

### CODE REVIEW

**Comment by MkMan** [comment]
Do you need to include the `dist` files? You might get merge conflicts on them.

File: src/components/AddToCartButton/AddToCartButton.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ const AddToCartButton = ({
     );
   }
 
-  if (!isInStock || !sku || !price) {
```
Comment (MkMan) [inline src/components/AddToCartButton/AddToCartButton.js:51]: Although unlikely but would this allow customers to add to cart even if a price isn't provided?

**Comment by kevin-ho87** [comment]
> Do you need to include the `dist` files? You might get merge conflicts on them.

This is to enable these Gel changes to appear in the 'web-ui' feature branch

File: src/components/AddToCartButton/AddToCartButton.js (hunk 1 above)
Comment (kevin-ho87) [reply src/components/AddToCartButton/AddToCartButton.js:51]: They won't be able to add to cart in this code block as this block handles the 'out of stock'/disabled state.

**Comment by andrwchang** [comment]
> > Do you need to include the `dist` files? You might get merge conflicts on them.
> 
> @MkMan This is to enable these Gel changes to appear in the 'web-ui' feature branch

We will need to come up with the better solution, prob run a build on the feature push. not sure if this will impact on the cost side FYI @maxlew

**Comment by maxlew** [comment]
> > > Do you need to include the `dist` files? You might get merge conflicts on them.
> > 
> > 
> > @MkMan This is to enable these Gel changes to appear in the 'web-ui' feature branch
> 
> We will need to come up with the better solution, prob run a build on the feature push. not sure if this will impact on the cost side FYI @maxlew

Yeah we need to come up with a better way of linking a feature branch into a gel PR or branch. Might be able to move the build to a 'post-install' step rather than having it commited as part of this repo. That way web-ui is responsible for handling that.

File: src/components/ProductGridItem/ProductGridItem.js [hunk 2]
```diff
@@ -19,74 +18,76 @@ import { P } from '~/components/Paragraph';
...
+
+    const classCtaLinkSet = cx(styles.addToCartButton, styles.ctaLink);
+
+    const RADIO_GROUP_NAME = sku;
+    const RADIO_GROUP_DATA_TEST_REF = 'PRODUCT_GRID_ITEM_VARIANT_SELECT';
+
+    return (
+      <div className={classSet} id={id} ref={ref}>
+        <Transition isActive={isImageActive} name="fade">
+          <Hyperlink className={styles.imageLink} url={url}>
```
Comment (zack76) [inline src/components/ProductGridItem/ProductGridItem.js:59]: just wonder if data-test-ref is needed for `Hyperlink` in this component

**Review by zack76 — APPROVED** [review]

File: src/components/ProductCommerce/ProductCommerce.module.css [hunk 3]
```diff
@@ -53,45 +61,79 @@
...
-    font-size: rem(14px);
+  @media (--viewport-lg) {
+    grid-column: 6/ 11;
   }
 }
 
-.variants {
-  justify-content: left;
-}
+@media all and (-ms-high-contrast: none) {
```
Comment (MkMan) [inline src/components/ProductCommerce/ProductCommerce.module.css:110]: Is this to target IE?

**Review by MkMan — APPROVED** [review]

**Review by maxlew — APPROVED** [review]

File: src/components/ProductGridItem/ProductGridItem.js (hunk 2 above)
Comment (kevin-ho87) [reply src/components/ProductGridItem/ProductGridItem.js:59]: I think it was only needed on the ATC button

File: src/components/ProductCommerce/ProductCommerce.module.css (hunk 3 above)
Comment (kevin-ho87) [reply src/components/ProductCommerce/ProductCommerce.module.css:110]: Yes that is correct, to fix the Grid layout on IE

**Pull request merged**